api/api/docs/base_docs.py

```diff
@@ -149,109 +140,64 @@ def build_source_path_parameter(media_type: MediaType):
-    """
-    elif collection == "source":
-        return f"""
-Get a collection of {media_type} from a specific source.
-
-This endpoint returns only the exact matches. To search within the source value,
-use the `search` endpoint with `source` query parameter.
-
-The results in the collection will be sorted by the order in which they
-were added to Openverse.
-    """
-    elif collection == "creator":
-        return f"""
-Get a collection of {media_type} by a specific creator from the specified source.
-
-This endpoint returns only the exact matches both on the creator and the source.
-Notice that a single creator's media items can be found on several sources, but
-this endpoint only returns the items from the specified source. To search within
-the creator value, use the `search` endpoint with `source` query parameter
-instead of `q`.
-
-The items will be sorted by the date when they were added to Openverse.
-    """
-
-
-COLLECTION_TO_OPERATION_ID = {
-    ("images", "source"): "images_by_source",
-    ("images", "creator"): "images_by_source_and_creator",
-    ("images", "tag"): "images_by_tag",
-    ("audio", "source"): "audio_by_source",
-    ("audio", "creator"): "audio_by_source_and_creator",
-    ("audio", "tag"): "audio_by_tag",
-}
-
-
-def collection_schema(
-    media_type: Literal["images", "audio"],
-    collection: Literal["source", "creator", "tag"],
-):
-    if media_type == "images":
-        request_serializer = PaginatedRequestSerializer
-        serializer = ImageSerializer
-    else:
-        request_serializer = AudioCollectionRequestSerializer
-        serializer = AudioSerializer
-
-    if collection == "tag":
-        responses = {
-            200: serializer(many=True),
-            404: NotFound,
-            400: ValidationError,
-            401: (NotAuthenticated, None),
-        }
-        path_parameters = [tag_path_parameter]
-    else:
-        responses = {
-            200: serializer(many=True),
-            404: source_404_response,
-            400: ValidationError,
-            401: (NotAuthenticated, None),
-        }
-        path_parameters = [build_source_path_parameter(media_type)]
-        if collection == "creator":
-            path_parameters.append(creator_path_parameter)
-    operation_id = COLLECTION_TO_OPERATION_ID[(media_type, collection)]
-    description = get_collection_description(media_type, collection)
-    return extend_schema(
-        operation_id=operation_id,
-        summary=operation_id,
-        auth=[],
-        description=description,
-        responses=responses,
-        parameters=[
-            request_serializer,
-            *path_parameters,
-        ],
-    )
+SEARCH_DESCRIPTION_DEFAULT = """
+Return audio files that match the query.
+
+This endpoint allows you to search within specific fields, or to retrieve
+a collection of all audio files from a specific source, creator or tag.
```

Comment:
Nit: should `audio` be here?
```suggestion
a collection of all %(media_type)s from a specific source, creator or tag.
```

Reply:
Do you mean `audio` instead of `audio files` ? I think we've talked about using audio vs audio files a couple of times, but I don't remember the conclusions.